data-collections-exercises.md

```diff
@@ -3,83 +3,98 @@
 Starting with the following array...
 
 ```rb
-planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
+# planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
 ```
 
 Access the second value in `planeteers`.
 
 ```rb
-# Your answer here
+# planeteers[1]
+------
+#  I think there is more than one way to do so 
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+# planeteers.push("Heart")
+-----
+# simliar to JS
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+# planeteers - ["Captain Planet"]
+----
 ```
 
 Combine `planeteers` with `rangers = ["Red", "Blue", "Pink", "Yellow", "Black"]` and save the result in a variable called `heroes`.
 
 ```rb
-# Your answer here
+# heros = rangers + planeteerss
+----
+
 ```
 
 Alphabetize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+#heroes.sort
 ```
 
 Randomize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+# heros.shuffle
+------
+# I think there is more than one way to do it.
 ```
 
 #### Bonus
 
 Select a random element from `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+# heros.sample
 ```
 
 Select all elements in `heroes` that begin with "B" using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+# planeteers.select{|word| word[0] === "B"}
 ```
 
 ### Hashes
 
 Initialize a hash called `ninja_turtle` with the properties `name`, `weapon` and `radical`. They should have values of "Michelangelo", "Nunchuks" and `true` respectively.
 
 ```rb
-# Your answer here
+# ninja_turtle = {
+#     name: "Michelangelo",
+#     weapon: "Nunchuks",
+#     radical: true,
+# }
 ```
 
 Add a key `age` to `ninja_turtle`. Set it to whatever numerical value you'd like.
 
 ```rb
-# Your answer here
+# ninja_turtle.merge({age:12})
+----
+# I used the merge method but I still think there is another way that I didn't figure out yet.
```

Comment:
when you use`merg` method it will add the age but it will create another copy of the hash in order to make changes directly in the hash we use `!`

```
ninja_turtle.merge!({age:12})
```